src/Microsoft.ML/Microsoft.ML.csproj

```diff
@@ -1,44 +1,24 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(RepoRoot)eng/pkg/Pack.props"/>
+<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <!-- To include the DLL and PDBs of the underlying projects
-    We need to remove the IncludeBuildOutput=false. This also generates a
-    Microsoft.ML.dll in the nuget package, and a Microsoft.ML.pdb in the .snupkg
-    But these are to be ignored, as they don't contain anything meaningful-->
-    <!-- <IncludeBuildOutput>false</IncludeBuildOutput> -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
```

Comment:
Did you need to set this to false? Should we upstream this change into the NoTargets SDK?

Reply:
Yeah, this needs to go in the NoTargets SDK.  Without it, this was trying to pack a non-existent file.

Reply:
Actually the default here would normally be correct.  This is necessary to override our repo's setting here: https://github.com/dotnet/machinelearning/blob/d0d8569220da60ff8acde3650393110c54dea8d0/src/Directory.Build.props#L6

Not sure if we should try bother upstreaming this.

Reply:
Consider conditioning this property on [`UsingMicrosoftNoTargetsSdk`](https://github.com/microsoft/MSBuildSdks/blob/1fd6d02672dc5d41cf498268a555b2f50f50f150/src/NoTargets/Sdk/Sdk.props#L10C6-L10C3) so that if more NoTargets projects get added, they don't have to repeat this setting.